roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -18,6 +18,8 @@ coredns_default_zone_cache_block: |
   cache 30
 
 coredns_pod_disruption_budget: false
+# when enable_dns_autoscaler is false, coredns_replicas is used to set the number of replicas
+coredns_replicas: 1
```

Comment:
HI @clwluvw 

Since Kubespray is designed for production environments, a default value of 2 would be more reasonable.

Reply:
Hi @yankay - Thanks for the review. It's now set to 2. Could you please take another look?

Reply:
Thanks @clwluvw 
/lgtm